README.md

```diff
@@ -120,3 +116,32 @@ Tweet at me! [@TheRealPomax](http://twitter.com/TheRealPomax) or [@TheRealPomax@
 ## And if I just want to use this?
 
 This code is [MIT licensed](https://raw.githubusercontent.com/Pomax/Font.js/master/LICENSE), do whatever you want with it.
+
+## NodeJS
+
+New to Font.js is the possibility to load the information of a font file located on your server, via NodeJS. Be sure to have the latest [NodeJS](https://nodejs.org/en/download/current/) installed, and run:
+```bash
+$ npm install
+```
+Create an example `font-node.js` file:
+```js
+import FontNode from './FontNode.js';
+
+const file = process.argv[2];
+if (file) {
+    new FontNode(file)
+        .then(font => {
+            console.log('*** FONT DATA ***');
```

Comment:
Maybe do an example here that's similar to the ones in the earlier, browser example?

For example

```javascript
   const GSUB = font.opentype.tables.GSUB;

    // Let's figure out which writing scripts this font supports:
    console.log(`This font supports the following scripts: ${
        `"${GSUB.getSupportedScripts().join(`", "`)}"`
    }`);
```